datashuttle/datashuttle.py

```diff
@@ -458,6 +458,47 @@ def show_configs(self):
         self.cfg.convert_str_and_pathlib_paths(copy_dict, "path_to_str")
         utils.message_user(json.dumps(copy_dict, indent=4))
 
+    def supply_config_file(
+        self, path_to_config: Union[Path, str], warn: bool = True
+    ):
+
+        path_to_config = Path(path_to_config)
+
+        utils.raise_error_not_exists_or_not_yaml(path_to_config)
+
+        if warn:
+            input_ = utils.get_user_input(
+                "This will overwrite the existing datashuttle config file."
```

Comment:
This warning also appears when one tries to create a new project with a self-supplied config file.
As far as I understand, you would only want the warning to appear if config file is supplied for an existing project.